spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/token/authentication/XsuaaJwtDecoderBuilder.java

```diff
@@ -30,6 +32,15 @@ public XsuaaJwtDecoderBuilder(XsuaaServiceConfiguration configuration) {
 	public JwtDecoder build() {
 		return new XsuaaJwtDecoder(configuration, decoderCacheValidity, decoderCacheSize, tokenValidators);
 	}
+	
+	/**
+	 * Assembles a ReactiveJwtDecoder
+	 *
+	 * @return ReactiveJwtDecoder
+	 */
+	public ReactiveJwtDecoder buildAsReactive() {
+		return new ReactiveXsuaaJwtDecoder(configuration, decoderCacheValidity, decoderCacheSize);
```

Comment:
When the builder is re-used (makes sense IMHO), the same fields must be provided. The reactive decoder does not have a field for oauth token validators. This is confusing for the user, because he can pass validators to the builder which will be ignored.

```suggestion
		return new ReactiveXsuaaJwtDecoder(configuration, decoderCacheValidity, decoderCacheSize, tokenValidators);
```